challenge.md

```diff
@@ -31,21 +31,17 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
- 
- <!-- Answer Ends Here -->
+ Absolute path
+
+
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
- 
- <!-- Answer Ends Here -->
+ Absolute path so that it directs specifically to that font.
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
- <!-- Answer Starts Here -->
- 
- <!-- Answer Ends Here -->
+ Absolue Path so that it doesnt bring up a random image.
```

Comment:
A relative path would be the right choice here. 

Why? Well, imagine that we had our project in a repo, and then we push that repo up to GitHub. What happens to a link that uses an absolute path? Well, GitHub doesn't have the same file structure as your computer, so when a file in your repo tries to find another file using that address, it can't - the result is a broken link. Using relative paths helps us avoid this.